underpants.sample.groups.json

```diff
@@ -0,0 +1,33 @@
+{
+  "host" : "underpants.company.com",
+  "oauth" : {
+    "domain"        : "company.com",
+    "client-id"     : "oauth-client-id",
+    "client-secret" : "oauth-client-secret"
+  },
+  "use-strict-security-headers": true,
+  "groups": {
+    "supersecret": [
+      "a@company.com",
+      "b@company.com"
+    ]
+  },
+  "certs" : [
+    {
+      "crt" : "/path/to/crt.pem",
+      "key" : "/path/to/key.pem"
+    }
+  ],
+  "routes" : [
+    {
+      "from"           : "public.company.com",
+      "to"             : "http://localhost:8080",
```

Comment:
Tracing through, let me know if any of this sounds wrong.

* `setup` parses the `To` URL and extracts the host and scheme, ignoring all else. (https://github.com/kellegous/underpants/blob/master/underpants.go#L418)
* the host and scheme are passed onto the `disp` object which handles requests for that route (https://github.com/kellegous/underpants/blob/master/underpants.go#L425)
* `serveHttpProxy` builds a new request to the by calling `urlFor` (https://github.com/kellegous/underpants/blob/master/underpants.go#L260)
* `urlFor` takes the requested URL and changes its host and scheme to those configured for the backend (https://github.com/kellegous/underpants/blob/master/underpants.go#L244-L247)

_so_, it's possible only if go's urlparser reserializes poorly.  whatcha think?